concepticondata/conceptlists/Lionnet-1985-364.tsv

```diff
@@ -0,0 +1,365 @@
+Lionnet-1985-364-284	284	pisar	tread	575	TREAD
+Lionnet-1985-364-285	285	pluma	feather	1201	FEATHER
+Lionnet-1985-364-286	286	poder	be able to	972	BE ABLE
+Lionnet-1985-364-287	287	podrirse, podre	rot	2907	ROT
+Lionnet-1985-364-288	288	prender lumbre	make fire	737	LIGHT (IGNITE)
+Lionnet-1985-364-289	289	pulga	flea	232	FLEA
+Lionnet-1985-364-290	290	punta	tip	2992	TIP (OF OBJECT)
+Lionnet-1985-364-291	291	quedarse, faltar	be left	987	REMAIN
+Lionnet-1985-364-292	292	quejarse	complain	3642	COMPLAIN
+Lionnet-1985-364-293	293	raíz	root	670	ROOT
+Lionnet-1985-364-294	294	rajarse	split	1437	SPLIT
+Lionnet-1985-364-295	295	rana	frog	503	FROG
+Lionnet-1985-364-296	296	raspar	scrape	1767	SCRAPE
+Lionnet-1985-364-297	297	rastrojo	stubble		
+Lionnet-1985-364-298	298	rata	rat	1490	RAT
+Lionnet-1985-364-299	299	ratón	mouse	1491	MOUSE
+Lionnet-1985-364-300	300	regalar	give gift		
+Lionnet-1985-364-301	301	regañar	scold	299	SCOLD
+Lionnet-1985-364-302	302	regresar	return	581	COME BACK
+Lionnet-1985-364-303	303	reírse	laugh	1355	LAUGH
+Lionnet-1985-364-304	304	renacuajo	tadpole		
+Lionnet-1985-364-305	305	robar	steal	713	STEAL
+Lionnet-1985-364-306	306	rodilla	knee	1371	KNEE
+Lionnet-1985-364-307	307	rojo	red	156	RED
+Lionnet-1985-364-308	308	saber	know	1410	KNOW (SOMETHING)
+Lionnet-1985-364-309	309	sal	salt	1274	SALT
+Lionnet-1985-364-310	310	salir	go out	1107	GO OUT
+Lionnet-1985-364-311	311	sangre	blood	946	BLOOD
+Lionnet-1985-364-312	312	sapo	toad	894	TOAD
+Lionnet-1985-364-313	313	secarse	to dry up	1786	DRY UP
+Lionnet-1985-364-314	314	seis	six	1703	SIX
+Lionnet-1985-364-315	315	sembrar	sow seeds	748	SOW SEEDS
+Lionnet-1985-364-316	316	semilla	seed	714	SEED
+Lionnet-1985-364-317	317	sentarse	sit down	1649	SIT DOWN
+Lionnet-1985-364-318	318	sentarse (suj. plur.)	sit down (several)		
+Lionnet-1985-364-319	319	sentir, desear	feel, desire	1079	FEEL
+Lionnet-1985-364-320	320	ser hondo	be deep	1593	DEEP
+Lionnet-1985-364-321	321	serpiente	snake	730	SNAKE
+Lionnet-1985-364-322	322	sol	sun	1343	SUN
+Lionnet-1985-364-323	323	soltarse	become loose, untie	1933	LOOSEN
+Lionnet-1985-364-324	324	sombra	shade, shadow	1388	SHADE
+Lionnet-1985-364-325	325	soplar	blow	176	BLOW (WITH MOUTH)
+Lionnet-1985-364-326	326	subir	go up, ascend	705	GO UP (ASCEND)
+Lionnet-1985-364-327	327	suegra	mother in-law	1050	MOTHER-IN-LAW
+Lionnet-1985-364-328	328	suegro	father in-law	1055	FATHER-IN-LAW
+Lionnet-1985-364-329	329	tabaco	tobacco	974	TOBACCO
+Lionnet-1985-364-330	330	tender	lay out horizontally, extend		
+Lionnet-1985-364-331	331	tener dificultad, ser flojo	have difficulties, be lazy		
+Lionnet-1985-364-332	332	tener miedo	be scared	1419	FEAR (BE AFRAID)
+Lionnet-1985-364-333	333	tener vergüenza	be ashamed		
+Lionnet-1985-364-334	334	teta	breast, teat	1402	BREAST
+Lionnet-1985-364-335	335	tierra	earth, soil	1228	EARTH (SOIL)
+Lionnet-1985-364-336	336	tierra de siembra	field for sowing	212	FIELD
+Lionnet-1985-364-337	337	tigre	tiger (jaguar)	846	TIGER
```

Comment:
Similar to Lion/Cougar, this seems to be a tiger/Jaguar. Even though it seems unintuitive, I guess we could even map this to Jaguar?